security/oc_pki.c

```diff
@@ -165,6 +162,8 @@ pki_add_identity_cert(size_t device, const unsigned char *cert,
   }
 
   /* Parse identity cert's private key */
+  mbedtls_pk_context pkey;
+  mbedtls_pk_init(&pkey);
```

Comment:
`mbedtls_pk_free(&pkey)` for error branch on line 170

Reply:
fixed